src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,49 @@
+package task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    private LocalDateTime deadline;
+
+    /**
+     * Constructs a <code>Deadline</code> object.
+     * @param description The description of the task with deadline
+     * @param deadline The deadline of the task
+     */
+    public Deadline(String description, LocalDateTime deadline) {
+
+        super(description);
+        this.deadline = deadline;
+    }
+
+    /**
+     * Constructs a <code>Deadline</code> object knowing it is already done.
+     * @param description The description of the task with deadline
+     * @param deadline The deadline of the task
+     * @param isDone Whether the deadline task is done or not
+     */
+    public Deadline(String description, LocalDateTime deadline, boolean isDone) {
+
+        super(description, isDone);
+        this.deadline = deadline;
+    }
+
+    @Override
+    public String toString() {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMMM dd yyyy, HH:mm");
+        return super.toString() + " (by: " + deadline.format(formatter) + ")";
+    }
+
+    @Override
+    public String simplifiedTaskString() {
```

Comment:
Names representing methods must be verbs and written in camelCase. e.g. getName(), computeTotalWidth()

Suggestion here would be to change it to getSimplifiedTaskString()

This minor infraction is similar across all objects extending from Task, so they could be refactored similarly

Reply:
That's right! Thanks for pointing that out